environment.yml

```diff
@@ -0,0 +1,28 @@
+# run: conda env create
```

Comment:
FWIW - we've always avoided putting a conda env in the main repo (we have one currently in the wiki only) because we haven't wanted to commit to keeping such a thing up to date. We have enough conda users in the group now that it's probably fine, but I wanted to at least flag this here.